lib/v2/dsndsht23/index.js

```diff
@@ -87,6 +87,19 @@ module.exports = async (ctx) => {
                         $(images[k]).replaceWith(`<img src="${$(images[k]).attr('file')}" />`);
                     }
                 }
+                if (images.length === 0) {
+                    const pattl = $('.pattl');
+                    const pattlImages = $(pattl).find('img');
+                    for (let k = 0; k < pattlImages.length; k++) {
+                        const file = $(pattlImages[k]).attr('file');
+                        if (!file || file === 'undefined') {
+                            $(pattlImages[k]).replaceWith('');
+                        } else {
+                            $(pattlImages[k]).replaceWith($(`<img src="${file}" />`));
+                        }
+                    }
```

Comment:
Please use for-of loop instead
https://rules.sonarsource.com/javascript/RSPEC-4138

Reply:
done